### PR TITLE
Create an FAQ section

### PR DESCRIPTION
**Pull request by ehumph**
This PR adds an FAQ section to the website. These FAQs have been gathered from the first few meetings with GDSCN faculty about the pilot analysis.

### CODE REVIEW

**Comment by ehumph** [comment]
@avahoffman I tried to be helpful and add the tab to the ui.R file, but now I have made the checks unhappy. It's specifically the code I added to ui.R causing the issue, but I don't see the problem. Any thoughts?

**Comment by avahoffman** [comment]
@ehumph as far as I know, this is a build check on the SciComp side of things. Let me run it locally and see what's up.

**Comment by avahoffman** [comment]
image
image

Looks like it has to do with the encoding. I'm going to open in Sublime Text and try to get rid of any special characters.

File: www/faq.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* Try to pick sites that you think would be interesting! For the pilot sampling, we focused on managed vs unmanaged sites and tried to sample a variety of landscapes. Managed sites are generally those that have been used as lawns, playing fields, or gardens; unmanaged sites include places like meadows and forests.
+* You should pick 5 sites, with 2 replicates at each, for a total of ten samples.
+
+
+## How do I get permission to sample at a location?
+Land that you might want to sample will generally be one of three types: public land (like parks or playing fields), university and college grounds, or private land.
+* For _public land_, the best approach is to email the appropriate contact person for permission. Public lands are generally managed by county, town, or city Park Services. Contact information can usually be found on the specific park websites. A template email is provided [here](https://docs.google.com/document/d/1C6bPBZFO2DhANMJHlNXqAMSAMBqpYjXlHl9JMxNAx6k/edit)!
+
+_Tip 1:_ Be specific about what sites you want to sample (they often will give a permit for just those sites).
```
Comment (avahoffman) [inline www/faq.md:17]: Let's break line between the tips

File: www/faq.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# FAQs
+
+If you don't find the answer to your question here, please contact Natalie Kucher at nkucher3@jhu.edu!
```
Comment (avahoffman) [inline www/faq.md:3]: Should we make this look like nkucher3 at jhu.edu ? So she doesn't get spammed?

File: www/faq.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+## How do I collect samples?
+We have prepared a protocol to make this easy! Check out the written protocol [here](https://docs.google.com/document/d/1oyFaknksSQEpwN7yGEBuGZaDGN96rKGOaovGsbhlGic/edit#heading=h.rlfcx57l2u2o). We have also created [YouTube videos](https://www.youtube.com/playlist?list=PLzgm426KgvrhheloBdlSWshM9v2VvJEcX) for you to reference.
+
+
+## What information do I need to collect from students before we go into the field?
+* Make sure you have all participating students sign a liability waiver. Your school may have a specific one they want you to use; check with administration or student services. 
+*If you are taking pictures for the GDSCN archives, we ask that you also have students sign a photo release form giving us permission to use their image on the website or in future presentations.
```
Comment (avahoffman) [inline www/faq.md:34]:
```suggestion
* If you are taking pictures for the GDSCN archives, we ask that you also have students sign a photo release form giving us permission to use their image on the website or in future presentations.
```

File: www/faq.md (hunk 2 above)
Comment (ehumph) [reply www/faq.md:3]: Yes, good idea!

**Pull request merged**